components/PhoneTrackingButton.vue

```diff
@@ -10,24 +10,31 @@ import Vue from "vue";
 
 export default Vue.extend({
   name: "PhoneTrackingButton",
-  data () {
+  data() {
     return {
       tracking: false,
-      watcherId: -1
+      watcherId: -1,
+      client: this.$initiateClient(),
```

Comment:
This creates a second MQTT client, right? As we already have 1 in `index.vue`, it thinks it's best that we just pass the client from there to this component and re-use it